src/Symfony/Component/Yaml/Inline.php

```diff
@@ -609,7 +609,11 @@ private static function evaluateScalar($scalar, $flags, $references = array())
             case $scalar[0] === '!':
                 switch (true) {
                     case 0 === strpos($scalar, '!str'):
+                        @trigger_error('Support for the !str tag is deprecated since version 3.4. Use the !!str tag instead.', E_USER_DEPRECATED);
+
                         return (string) substr($scalar, 5);
+                    case 0 === strpos($scalar, '!!str '):
+                        return (string) substr($scalar, 6);
```

Comment:
if we use parseScalar, we may end up parsing the value as a YAML timestamp for instance, which will break the usage of the tag (see `not-date: !!str 2002-04-28` in the tests for instance)